distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Normalization.enso

```diff
@@ -0,0 +1,54 @@
+import project.Any.Any
+
+import project.Data.Vector.Vector
+import project.Metadata.Display
+import project.Metadata.Widget
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Single_Choice
+
+polyglot java import com.ibm.icu.text.Normalizer2.Mode
+
+## Enso representations of `com.ibm.icu.text.Normalizer2.Mode`. Does not
+   include all values of `Mode`.
+type Normalization
+    ## Unicode NFC normalization mode.
+    NFC
+    ## Unicode NFD normalization mode.
+    NFD
+    ## Unicode NFKC normalization mode.
+    NFKC
+    ## Unicode NFKD normalization mode.
+    NFKD
+    ## Unicode NFKC_CF normalization mode.
+    NFKCCasefold
```

Comment:
Personal preference but nice to have a blank line between each.

Reply:
Done